### PR TITLE
functionality for getting all reviews added

### PR DESCRIPTION
**Pull request by melsem08**

### CODE REVIEW

File: src/components/ReviewsList.jsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import { useState, useEffect } from "react";
+import { Link } from "react-router-dom";
+import { getReviews } from "../../utils";
+
+export function ReviewsList() {
+  const [reviews, setReviews] = useState([]);
```
Comment (Keb84) [inline src/components/ReviewsList.jsx:6]: some state for loading would be good here 👀 to give the use some visual feedback while the page is rendering

File: src/components/ReviewsList.jsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+      <h2>Reviews</h2>
+      <ul>
+        {reviews.map((review) => {
+          return (
+            <li key={review.review_id}>
+              <Link to={`/${review.review_id}`}>
+                <h3>{review.title}</h3>
+              </Link>
+              <p>by {review.designer}</p>
+            </li>
```
Comment (Keb84) [inline src/components/ReviewsList.jsx:35]: I would consider also displaying the image for each review 💭

File: utils.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
+export function getReviews() {
+  return fetch(`https://games-api-project.onrender.com/api/reviews`).then(
+    (res) => {
+      return res.json();
+    }
```
Comment (Keb84) [inline utils.js:5]: Great that you've put your api calls into a utils file 🤸

**Review by Keb84 — CHANGES_REQUESTED** [review]
Great work on this ticket so far 🎉 please have a look at the suggestions and resubmit a PR 🙂

**Review by MrMcSnail — APPROVED** [review]
Looks good! Merge away!

**Pull request merged**